packages/blockchain-wallet-v4-frontend/src/scenes/Settings/General/LinkedBanks/template.success.tsx

```diff
@@ -0,0 +1,134 @@
+import { Button, Icon, Text } from 'blockchain-info-components'
+import { convertBaseToStandard } from 'data/components/exchange/services'
+import { fiatToString } from 'core/exchange/currency'
+import { FormattedMessage } from 'react-intl'
+import { InjectedFormProps, reduxForm } from 'redux-form'
+import { Props as OwnProps, SuccessStateType } from '.'
+import {
+  SettingContainer,
+  SettingHeader,
+  SettingSummary
+} from 'components/Setting'
+import media from 'services/ResponsiveService'
+import React from 'react'
+import styled from 'styled-components'
+
+const CardWrapper = styled.div`
```

Comment:
If you're copy and pasting things from Cards please create a shared styled component.